### PR TITLE
toss everything deprecated

### PR DESCRIPTION
**Pull request by ix0rai**
removes the last of the deprecated things. one thing I'm not sure about removing is the `getSource` overload, I don't see much of a point to removing it

### CODE REVIEW

File: enigma/src/main/java/org/quiltmc/enigma/source/Decompiler.java [hunk 1]
```diff
@@ -4,10 +4,5 @@
 import org.checkerframework.checker.nullness.qual.Nullable;
 
 public interface Decompiler {
-	@Deprecated // use remapper specific one for easy doc inclusion
-	default Source getSource(String className) {
```
Comment (IotaBread) [inline enigma/src/main/java/org/quiltmc/enigma/source/Decompiler.java:8]: Id keep this method, or rename it to `getUndocumentedSource`

Comment (ix0rai) [reply enigma/src/main/java/org/quiltmc/enigma/source/Decompiler.java:8]: that's what I was thinking!

File: enigma/src/main/java/org/quiltmc/enigma/source/Decompiler.java [hunk 2]
```diff
@@ -4,10 +4,9 @@
...
-		return this.getSource(className, null);
+	default Source getUndocumentedSource(String className) {
+		return this.getUndocumentedSource(className, null);
 	}
 
-	Source getSource(String className, @Nullable EntryRemapper remapper);
+	Source getUndocumentedSource(String className, @Nullable EntryRemapper remapper);
```
Comment (IotaBread) [inline enigma/src/main/java/org/quiltmc/enigma/source/Decompiler.java:11]: This one does produce documented source :p

Comment (IotaBread) [reply enigma/src/main/java/org/quiltmc/enigma/source/Decompiler.java:11]: At least if remapper isn't null

Comment (ix0rai) [reply enigma/src/main/java/org/quiltmc/enigma/source/Decompiler.java:11]: throwing up how did I not notice that happened :p

Comment (ix0rai) [reply enigma/src/main/java/org/quiltmc/enigma/source/Decompiler.java]: ah intellij overload renaming
fixed!

**Review by IotaBread — APPROVED** [review]

**Pull request merged**